src/js/background/context-menu.js

```diff
@@ -217,12 +217,19 @@ const relayContextMenus = {
       // Loop Through Existing Aliases
       if (opts?.createExistingAliases) {
         
+        const shouldAliasOrderBeReversed = await getServerStoragePref();
```

Comment:
Could you add a comment here about _why_ the order should be revered if the labels are stored on the server? That's not clear to me :/

Reply:
Added in 54a57e0b85ff4b99e59537b9d7a319f25cb07bb2